docs/design/keyboard-shortcuts.md

```diff
@@ -150,26 +150,26 @@ Custom keyboard shortcuts are defined in a JSON file. This file describes your k
 
 Use the following guidelines to create custom key combinations for your add-ins.
 
-- A keyboard shortcut must include at least one modifier key (Alt/Option, Ctrl/Command, Shift) and only one other key. These keys must be joined with a `+` character.
-- The Command modifier key is supported on the macOS platform.
-- On macOS, the Alt key is mapped to the Option key. On Windows, the Command key is mapped to the Ctrl key.
-- The Shift key can't be used as the only modifier key. It must be combined with either Alt/Option or Ctrl/Command.
+- A keyboard shortcut must include at least one modifier key (<kbd>Alt</kbd>/<kbd>Option</kbd>, <kbd>Ctrl</kbd>/<kbd>Command</kbd>, <kbd>Shift</kbd>) and only one other key. These keys must be joined with a `+` character.
+- The <kbd>Command</kbd> modifier key is supported on the macOS platform.
```

Comment:
Later in this PR, you refer to the Command key as "Cmd". Is there a standard?

Reply:
We've been using [VS Code notations](https://code.visualstudio.com/docs/getstarted/keybindings#_accepted-keys) and that seems to be using Cmd so I've updated to that